eventmesh-storage-plugin/eventmesh-storage-pulsar/src/main/java/org/apache/eventmesh/storage/pulsar/consumer/PulsarConsumerImpl.java

```diff
@@ -140,6 +153,13 @@ public void commit(EventMeshAction action) {
                             String.format("Failed to unsubscribe the topic:%s with exception: %s", subTopic, ex.getMessage()));
                     } catch (EventDeserializationException ex) {
                         log.warn("The Message isn't json format, with exception:{}", ex.getMessage());
+                    } catch (Exception e) {
+                        ackConsumer.negativeAcknowledge(msg);
+                        try {
+                            ackConsumer.reconsumeLater(msg, 5, TimeUnit.SECONDS);
```

Comment:
For `negativeAcknowledge()` and `reconsumeLater()`, the former will automatically re-consume messages from the retry topic `retryTopic`, while the latter will re-consume messages from the current topic `subTopic`. Why do you need to use both? Have I misunderstood?

Reply:
The negativeAcknowledge() method enables the redelivery of failed messages by sending them back to the main queue. The reconsumeLater() method, on the other hand, handles retries by moving the failed messages to a special retry topic stored in a retry queue, where they will be reprocessed after a specified delay.

Reply:
Compared with negative acknowledgment, retry letter topic is more suitable for messages that require a large number of retries with a configurable retry interval.The messages in the retry letter topic are persisted to BookKeeper, while messages that need to be retried due to negative acknowledgment are cached on the client side.Negative acks and retry/DLQ do work together

Reply:
> Negative acks and retry/DLQ do work together

Are you sure it's a collaboration and not duplicate consumption? If you are sure, could you explain this: how can it ensure that only one of the mechanisms will be triggered for retries when there are failed messages, given that both mechanisms exist simultaneously?

Reply:
Sorry, I had a misunderstanding. The negative acknowledgment will cause duplicate consumption with active retry. I have fixed it.